libr/core/cmd_flag.c

```diff
@@ -420,6 +421,47 @@ static int flag_to_flag(RCore *core, const char *glob) {
 	return 0;
 }
 
+typedef struct {
+	RTable *t;
+} FlagTableData;
+
+static bool __tableItemCallback(RFlagItem *flag, void *user) {
+	FlagTableData *ftd = user;
+	if (flag->name && *flag->name) {
```

Comment:
There is already a macro for this. It should be called R_STR_EMPTY or something like that